### ISSUE TITLE
Add info to README re: other feltr package

### ISSUE THREAD

**Issue by elipousson** [christopherkenny/feltr#2]
Hey – I just stumbled across your blog post about this new package but was sorry to see that we landed on the same name for an R package with the same purpose. I have not submitted to CRAN but started working on my package back in April. To avoid any confusion for users, I'd be happy to add a note to my README referencing your package and warning about possible name conflicts and would appreciate if you could do the same.

**Comment by christopherkenny** [comment]
Hi, thanks for reaching out. Sorry for any confusion, I was unaware of your package. Happy to add a note to the readme with a link to your project. Beware that my package is on [CRAN](https://cran.r-project.org/web/packages/feltr/index.html), so I'm locked into the name.

**Comment by christopherkenny** [comment]
Added a note! https://github.com/christopherkenny/feltr#related-packages

**Comment by elipousson** [comment]
Thanks! The name is clearly the right choice so I can definitely understand how this could happen. I also added a link to your package from my README. I'll let you know if I decide to rename and submit to CRAN but since you have the full range of API endpoints covered I don't think I'm likely to make it a priority.

If you were interested in adding URL parsing for the map_id parameter feel free to borrow from [my helper function](https://github.com/elipousson/feltr/blob/main/R/felt_url.R#L36-L40). I've also added a variety of alert messages to confirm map or layer creation that may be helpful to consider incorporating.
